webpack.config.js

```diff
@@ -46,11 +46,22 @@ module.exports = {
       template: path.resolve('./', 'index.html'),
       filename: 'index.html',
       hash: true
-    })
+    }),
+    new webpack.EnvironmentPlugin(
+      [
+        'SPOOF_SINOPIA_SERVER',
```

Comment:
This tells webpack at build time to set replace all instances of `process.env.{WHATEVER}` in the code with values from the build environment.